cli/cook/subcommands/usage.py

```diff
@@ -127,53 +164,101 @@ def format_percent(n):
-            share_map = cluster_usage['share']
-            print_info(colors.bold(cluster))
-            print_info(format_share(share_map))
-            print_info(format_usage(usage_map))
-            applications = cluster_usage['applications']
-            if applications:
-                print_info('Applications:')
+        if 'using_pools' in cluster_usage:
+            if cluster_usage['using_pools']:
+                for pool, pool_usage in cluster_usage['pools'].items():
+                    state = ' (inactive)' if pool_usage['state'] == 'inactive' else ''
+                    print_formatted_cluster_or_pool_usage(f'{cluster} - {pool}{state}', pool_usage)
             else:
-                print_info(colors.waiting('Nothing Running'))
-            for application, application_usage in applications.items():
-                usage_map = application_usage['usage']
-                print_info(f'- {colors.running(application if application else "[no application defined]")}')
-                print_info(f'  {format_usage(usage_map)}')
-                print_info('  Job Groups:')
-                for group, group_usage in application_usage['groups'].items():
-                    usage_map = group_usage['usage']
-                    jobs = group_usage['jobs']
-                    print_info(f'\t- {colors.bold(group if group else "[ungrouped]")}')
-                    print_info(f'\t  {format_usage(usage_map)}')
-                    print_info(f'\t  Jobs: {len(jobs)}')
-                    print_info('')
-            print_info('')
+                print_formatted_cluster_or_pool_usage(cluster, cluster_usage)
 
+def filter_query_result_by_pools(query_result, pools):
+    """Filter query result if pools are provided. Return warning message if some of the pools not found in any cluster"""
+
+    clusters = []
+    known_pools = []
+
+    pools_set = set(pools)
+    filtered_clusters = {}
+    for cluster, cluster_usage in query_result['clusters'].items():
+        clusters.append(cluster)
+        if cluster_usage['using_pools']:
+            filtered_pools = {}
+            for pool, pool_usage in cluster_usage['pools'].items():
+                known_pools.append(pool)
+                if pool in pools_set:
+                    filtered_pools[pool] = pool_usage
+            if filtered_pools:
+                filtered_clusters[cluster] = cluster_usage
+                cluster_usage['pools'] = filtered_pools
+    query_result['clusters'] = filtered_clusters
+
+    missing_pools = pools_set.difference(set(known_pools))
+    if missing_pools:
+        print_error(f"WARNING: " +
+                    (f"'{list(missing_pools)[0]}' is not a valid pool in "
+                     if len(missing_pools) == 1 else
+                     f"{list(missing_pools)} are not valid pools in ") +
+                    (f"cluster '{clusters[0]}'"
+                     if len(clusters) == 1 else
+                     f"clusters {clusters}"))
```

Comment:
I think we should make this look more like the error message you get from `jobs` when you don't get any results:

`No matching running jobs for dpo found A / B / C / D / E.`

Specifically:

- remove "WARNING" since it already shows up in red
- remove "clusters"
- remove the brackets and single quotes around the clusters and separate with ` / `
- add a period at the end of the sentence